msctl

```diff
@@ -2872,9 +2872,11 @@ case "$COMMAND" in
     # Rename the world.
     printf "Renaming Minecraft world: $1"
     mkdir -p "$WORLDS_LOCATION/$2"
-    mv $WORLDS_LOCATION/$1 $WORLDS_LOCATION/$2
+    mv $WORLDS_LOCATION/$1/* $WORLDS_LOCATION/$2
+    rm -r $WORLDS_LOCATION/$1
     if [ -d $WORLDS_LOCATION/$2/$1 ]; then
-      mv $WORLDS_LOCATION/$2/$1 $WORLDS_LOCATION/$2/$2
+      mv $WORLDS_LOCATION/$2/$1/* $WORLDS_LOCATION/$2/$2
+      rm -r $WORLDS_LOCATION/$2/$1
```

Comment:
I do not see the difference in the rename process here
Unless creating the directory before the mv command is what is making it fail.
The 2nd command to mv the world directory inside the main world folder shouldn't have the same issue since the directory wouldn't already exist.

Reply:
With the old command, a hypothetical file `$WORLDS_LOCATION/$1/file.txt` would end up in `$WORLDS_LOCATION/$2/$1/file.txt`, since mv moves the whole directory inside the new one.

See this as an example:

```
❯ mkdir old; touch old/file; mkdir -p new
❯ mv -v /tmp/old /tmp/new
renamed '/tmp/old' -> '/tmp/new/old'
❯ ls -la /tmp/new
total 12
drwxrwxr-x  3 xathon xathon 4096 Apr 16 00:30 .
drwxrwxrwt 17 root   root   4096 Apr 16 00:30 ..
drwxrwxr-x  2 xathon xathon 4096 Apr 16 00:30 old
❯ ls -la /tmp/new/old
total 8
drwxrwxr-x 2 xathon xathon 4096 Apr 16 00:30 .
drwxrwxr-x 3 xathon xathon 4096 Apr 16 00:30 ..
-rw-rw-r-- 1 xathon xathon    0 Apr 16 00:30 file
```

What my code does is moving everything inside the old directory inside the new one, then deleting the (now empty) old one.

Though, I just realized I missed creating a directory in the 2nd command...